torch_geometric/nn/conv/gprgnn.py

```diff
@@ -0,0 +1,166 @@
+from typing import Optional
+
+import numpy as np
+import torch.nn.functional as F
+from torch import Tensor
+from torch.nn import Parameter
+
+from torch_geometric.nn.conv import MessagePassing
+from torch_geometric.nn.conv.gcn_conv import gcn_norm
+from torch_geometric.typing import Adj, OptPairTensor, OptTensor, SparseTensor
+from torch_geometric.utils import spmm
+
+
+class GPRGNN(MessagePassing):
```

Comment:
No strong opinion. I think we should name it something like `GPRConv` rather than `GPRGNN` as it sounds like a model instead of a layer.

Reply:
I think it would be better to keep the same name as in the paper and their original implementation. Would keep the name for now.